distributed/deploy/tests/test_local.py

```diff
@@ -1045,3 +1045,10 @@ async def test_no_workers(cleanup):
         n_workers=0, silence_logs=False, dashboard_address=None, asynchronous=True
     ) as c:
         pass
+
+
+@pytest.mark.asyncio
+async def test__default_scheduler_port():
+    async with LocalCluster(n_workers=2, processes=False, asynchronous=True) as cluster:
+
+        assert cluster.scheduler_port == 8786
```

Comment:
`LocalCluster` doesn't have a `scheduler_port` attribute. Though it does have a `scheduler_address` attribute from which we can extract the port using `parse_host_port` in `distributed/comm/addressing.py` with something like 

```diff
diff --git a/distributed/deploy/tests/test_local.py b/distributed/deploy/tests/test_local.py
index 79ae3a7f..fc034ab0 100644
--- a/distributed/deploy/tests/test_local.py
+++ b/distributed/deploy/tests/test_local.py
@@ -16,6 +16,7 @@

 from dask.system import CPU_COUNT
 from distributed import Client, Worker, Nanny, get_client
+from distributed.comm.addressing import parse_host_port
 from distributed.core import Status
 from distributed.deploy.local import LocalCluster
 from distributed.metrics import time
@@ -1050,5 +1051,5 @@ async def test_no_workers(cleanup):
 @pytest.mark.asyncio
 async def test__default_scheduler_port():
     async with LocalCluster(n_workers=2, processes=False, asynchronous=True) as cluster:
-
-        assert cluster.scheduler_port == 8786
+        _, port = parse_host_port(cluster.scheduler_address)
+        assert port == 8786
```

Reply:
Oops, I forgot to push the changes that I've done to that test, but your suggestion looks way better than what I had (I was checking if 8786 were in `cluster.scheduler_address` 😬  I'll update this PR in a second